tests/test_api_client.py

```diff
@@ -197,16 +205,34 @@ async def test_not_possible_to_reauth(httpx_mock):
         status_code=401,
         content=b"unauth!",
     )
-    httpx_mock.add_response(
-        url="https://elis.rossum.ai/api/v1/auth/login",
-        status_code=400,
-        json={"password": ["This field may not be blank."]},
-    )
     with pytest.raises(APIClientError) as err:
         await client.fetch_one(Resource.User, 1)
 
-    assert err.value.status_code == 400
-    assert err.value.error == '{"password": ["This field may not be blank."]}'
+    assert err.value.status_code == 401
+
+
+@pytest.mark.asyncio
+async def test_reauth_success(login_mock):
+    """Invalid token used, credentials available => reauthenticate in the background. Fetch user without raising any errors."""
+    client = APIClient("username", "password", token=FAKE_TOKEN)
+
+    login_mock.add_response(
```

Comment:
```suggestion
    httpx_mock.add_response(
```

I would prefer to use the `httpx_mock` for mocking these GET user requests. Using the `login_mock` fixture is a bit confusing given the name.

Reply:
sure, why not. We still need the `login_mock` but we can have them both. Updated